recipes/4ti2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "4ti2" %}
+{% set version = "1.6.9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/4ti2/4ti2/releases/download/Release_{{ version.replace(".", "_") }}/4ti2-{{ version }}.tar.gz
+  sha256: 3053e7467b5585ad852f6a56e78e28352653943e7249ad5e5174d4744d174966
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage("4ti2") }}
```

Comment:
Why pin this? This package has only binaries right?

Reply:
You're right of course. I got confused by the `lib` directory.